iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/methods/DeviceMethodTests.java

```diff
@@ -352,62 +201,6 @@ public void invokeMethodRecoverFromTimeoutSucceed() throws Exception
         Assert.assertEquals(buildExceptionMessage("Unexpected status errors occurred", testInstance.deviceTestManager.client), 0, deviceTestManger.getStatusError());
     }
 
-    @Test
-    @StandardTierHubOnlyTest
-    @ContinuousIntegrationTest
-    public void invokeMethodDefaultResponseTimeoutSucceed() throws Exception
```

Comment:
Not worth doing this test since the only difference from the basic invokeMethod test is that it has a custom connect timeout set.

Reply:
Is custom connect timeout tested elsewhere?